frontend/src/plugin/index.ts

```diff
@@ -8,7 +8,6 @@ import * as ReactMonacoEditor from '@monaco-editor/react';
 import * as MuiLab from '@mui/lab';
 import * as MuiMaterial from '@mui/material';
 import * as MuiMaterialStyles from '@mui/material/styles';
-import * as MuiStyles from '@mui/styles';
```

Comment:
This removal could be done in a separate commit/PR.

Reply:
since MuiStyles doesn't work with react 18, I think it makes sense that it's removed in a commit together with upgrade to react 18